cmd/nvidia-cdi-hook/main.go

```diff
@@ -44,10 +45,10 @@ func main() {
 	opts := options{}
 
 	// Create the top-level CLI
-	c := cli.NewApp()
+	c := cli.Command{}
```

Comment:
Given that we're now constructing the command directly, let's set the members inline:
```
c := cli.Command{
  Name: "NVIDIA CDI Hook"
  ...
  Action: func(...)
```

Reply:
done